src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--mon calendar--month-length--31">
+      <div class="calendar__cell calendar__cell--1"></div>
```

Comment:
there is no need to set additional class with number of day
```suggestion
      <div class="calendar__cell"></div>
```